src/util/requests.js

```diff
@@ -43,15 +43,15 @@ export const sanitizeSnippetList = (snippetList) => {
   return snippetList;
 };
 
-export const makeUserUrl = user => `${GITHUB_API_URL}/users/${user}`;
+export const makeUserUrl = () => `${GITHUB_API_URL}/user`;
```

Comment:
You can't do this because in this context `user` isn't necessarily the authenticated user, can also be the owner of the snippet.